### PR TITLE
Add Arduino sample documentation

### PR DESCRIPTION
**Pull request by Mandur**
# PR for issue #1013

## What is being addressed

Adding documentation on the arduino required changes. 
If there is any feedback there, I will adress it after my vacations.

### CODE REVIEW

File: docs/tools/arduino-samples.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# Arduino Samples
+
+Arduino samples can be found on the Arduino folder in the repository root. This code is built to be used for the [Seeduino LoRaWan](https://wiki.seeedstudio.com/Seeeduino_LoRAWAN/) but can serve as reference for other platforms. We have region based examples for OTAA and ABP scenarios. Please follow the [documentation](https://wiki.seeedstudio.com/Seeed_Arduino_Boards/) to set up the device
+
+## Accessing the library files locally
+
+There are multuple scenarios where we would want to have the librairies be modified (listed here below). To access the local installation open your arduino IDE, go to File -> Preferences and click on the line after "more preference can be edited directly in the file".
```
Comment (roel4ez) [inline docs/tools/arduino-samples.md:7]:
```suggestion
There are multiple scenarios where we would want to have the libraries be modified (listed here below). To access the local installation, open your Arduino IDE, go to File -> Preferences and click on the line after "more preference can be edited directly in the file".
```

File: docs/tools/arduino-samples.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+![arduino files location](../images/arduinolibrarycustomsetup.png)
+
+Go next to the path "packages/Seeeduino/hardware/samd/1.8.2/libraries/LoRaWan and open LoRaWan.cpp and LoRaWan.h in a text editor.
```
Comment (roel4ez) [inline docs/tools/arduino-samples.md:11]:
```suggestion
Go next to the path `packages/Seeeduino/hardware/samd/1.8.2/libraries/LoRaWan` and open `LoRaWan.cpp` and `LoRaWan.h` in a text editor.
```

File: docs/tools/arduino-samples.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+
+### CN library changes
+
+To run our China samples some changes to the library need to be done as described below.
+
+paste the following in the LoRaWan.h file:
+
+``` C
+        void setChannelON(unsigned char channel);
```
Comment (roel4ez) [inline docs/tools/arduino-samples.md:21]:
```suggestion
void setChannelON(unsigned char channel);
```

File: docs/tools/arduino-samples.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+        void setChannelOFF(unsigned char channel);
```
Comment (roel4ez) [inline docs/tools/arduino-samples.md:22]:
```suggestion
void setChannelOFF(unsigned char channel);
```

**Review by roel4ez — CHANGES_REQUESTED** [review]
There is already a short document on the Arduino Sample: https://azure.github.io/iotedge-lorawan-starterkit/dev/samples/arduino/

Not sure if that one should be extended, or a new article should be created. Could be under the same `Samples` directory.

Also, you'll need to add the file to `mkdocs.yml` > `nav` section to make it show up in the docs.

File: docs/tools/arduino-samples.md (hunk 2 above)
Comment (Mandur) [reply docs/tools/arduino-samples.md]: Thank you for the suggestions!

**Comment by roel4ez** [comment]
You'll need to add the file to mkdocs.yml > nav section to make it show up in the docs. Files are not added automatically

**Review by danigian — APPROVED** [review]
Approving for what it concerns the 'docs/tools/arduino-samples.md' file.

Please have a look at merge conflicts and what Roel wrote about mkdocs.yml

**Review by roel4ez — APPROVED** [review]

**Pull request merged**